ConsoleApp/Vote/Vote.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp.Vote
+{
+    internal class Vote
```

Comment:
it should be not only yes/no vote app, it should have other options that user will provide